napari/builtins.yaml

```diff
@@ -7,57 +7,336 @@ private: true
+    - id: napari_builtins.data.moon
+      title: Generate moon sample
+      python_name: napari.plugins._skimage_data:moon
+    - id: napari_builtins.data.page
+      title: Generate page sample
+      python_name: napari.plugins._skimage_data:page
+    - id: napari_builtins.data.retina
+      title: Generate retina sample
+      python_name: napari.plugins._skimage_data:retina
+    - id: napari_builtins.data.rocket
+      title: Generate rocket sample
+      python_name: napari.plugins._skimage_data:rocket
+    - id: napari_builtins.data.shepp_logan_phantom
+      title: Generate shepp_logan_phantom sample
+      python_name: napari.plugins._skimage_data:shepp_logan_phantom
+    - id: napari_builtins.data.skin
+      title: Generate skin sample
+      python_name: napari.plugins._skimage_data:skin
+    - id: napari_builtins.data.text
+      title: Generate text sample
+      python_name: napari.plugins._skimage_data:text
 
   readers:
-  - command: napari_builtins.get_reader
-    accepts_directories: true  
-    filename_patterns: ['*.csv','*.npy']
+    - command: napari_builtins.get_reader
+      accepts_directories: true
+      filename_patterns: ["*.csv", "*.npy"]
 
   writers:
-  - command: napari_builtins.write_image
-    filename_extensions: ['.tif','.tiff','.bmp','.bsdf','.bw','.eps','.gif','.icns','.ico','.im','.lsm','.npz','.pbm','.pcx','.pgm','.png','.ppm','.ps','.rgb','.rgba','.sgi','.stk','.tga']
-    layer_types: ['image']
-    name: lossless
-
-  - command: napari_builtins.write_image
-    filename_extensions: ['.j2c','.j2k','.jfif','.jp2','.jpc','.jpe','.jpeg','.jpf','.jpg','.jpx','.mpo']
-    layer_types: ['image']
-    name: lossy
-
-  - command: napari_builtins.write_labels
-    filename_extensions: ['.tif','.tiff','.bsdf','.im','.lsm','.npz','.pbm','.pcx','.pgm','.ppm','.stk']
-    layer_types: ['labels']
-    name: labels
-
-  - command: napari_builtins.write_points
-    filename_extensions: ['.csv']
-    layer_types: ['points']
-    name: points
-
-  - command: napari_builtins.write_shapes
-    filename_extensions: ['.csv']
-    layer_types: ['shapes']
-    name: shapes
-  
-  - command: napari_builtins.write_directory
-    layer_types: ['image*','labels*','points*','shapes*']
-    name: Save to Folder
\ No newline at end of file
+    - command: napari_builtins.write_image
+      filename_extensions:
+        [
+          ".tif",
+          ".tiff",
+          ".bmp",
+          ".bsdf",
+          ".bw",
+          ".eps",
+          ".gif",
+          ".icns",
+          ".ico",
+          ".im",
+          ".lsm",
+          ".npz",
+          ".pbm",
+          ".pcx",
+          ".pgm",
+          ".png",
+          ".ppm",
+          ".ps",
+          ".rgb",
+          ".rgba",
+          ".sgi",
+          ".stk",
+          ".tga",
+        ]
+      layer_types: ["image"]
+      name: lossless
+
+    - command: napari_builtins.write_image
+      filename_extensions:
+        [
+          ".j2c",
```

Comment:
Would you move .jpg to the front of the list.  The first entry in this list ends up being the default selection when a user clicks on the entry in the file save dialog; j2c shouldn't be the default.  Maybe I should make a separate pr, but since it's getting changed now...

Reply:
sorry, this was all just autoformatting... will change back

Reply:
> Would you move .jpg to the front of the list.

sure though :)

Reply:
Thanks!  I keep fighting w the autoformatter here too.  I'm happy letting it win this one :)